### PR TITLE
Create a symlink from workspace .bloop to out .bloop for BloopP…

### PR DESCRIPTION
**Pull request by wiwa**
This will allow users to use `bloop` commands (e.g. `bloop test ...`) directly from their workspace.

### CODE REVIEW

File: metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala [hunk 1]
```diff
@@ -218,6 +213,19 @@ object BloopPants {
...
   }
 
+  private def symlinkToOut(args: Args): Unit = {
+    val workspaceBloop = args.workspace.resolve(".bloop")
+
+    if (!Files.exists(workspaceBloop) || Files.isSymbolicLink(workspaceBloop)) {
+      val outBloop = args.out.resolve(".bloop")
+      val actionWord =
+        if (Files.isSymbolicLink(workspaceBloop)) "overwriting" else "creating"
+      scribe.info(s"$actionWord symlink from $workspaceBloop to $outBloop")
```
Comment (olafurpg) [inline metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala]: Let's drop this logging statement

File: metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala [hunk 2]
```diff
@@ -1,38 +1,32 @@
 package scala.meta.internal.pantsbuild
 
 import bloop.config.{Config => C}
-import java.nio.file.Paths
-import java.nio.file.Files
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
```
Comment (olafurpg) [inline metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala]: Can you please configure IntelliJ to 

* sort everything alphabetically
* don't group prefix with `import a.{b,c}`

**Comment by olafurpg** [comment]
we can ignore the unrelated CI failure, it should no longer be a problem after we merge #1277

File: metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala (hunk 2 above)
Comment (wiwa) [reply metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala]: Done.

File: metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala (hunk 1 above)
Comment (wiwa) [reply metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala]: done

**Pull request merged**